python-lib/hubspot/companies.py

```diff
@@ -4,34 +4,36 @@
 import json, time, requests, logging
 import urllib
 from pandas.io.json import json_normalize
+from constants import Constants
+
+const = Constants()
```

Comment:
Constants.COMPANIES_LIMIT is static, so you don't need to declare a variable.
Furthermore, "const" is a reserved keyword in a lot of languages (Java, C, C++...) so it must be avoided.

Reply:
Good to know. Changed it. Thanks Mayeul!